WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -879,14 +879,15 @@ private extension NotificationsViewController {
             !$0.read
         }
 
-        NotificationSyncMediator()?.markAsRead(unreadNotifications, completion: { error in
+        NotificationSyncMediator()?.markAsRead(unreadNotifications, completion: { [weak self] error in
+            guard let self = self else { return }
```

Comment:
We probably should always send a notice. This will prevent that. Let's not do this, and call `updateMarkAllAsReadButton` with `self?.`.

Reply:
Wouldn't it mean that we don't have the NotificationsViewController anymore? Would we want to send the notice on another screen?

Reply:
Yea, the user might have moved on. However, we still display notices so they still get confirmation their action was successful.

Reply:
I understand. Updating as `self?` then